src/onegov/ballot/models/election/election.py

```diff
@@ -345,8 +347,9 @@ def export(self):
         rows = []
         for result in results:
             row = OrderedDict()
-            for locale, title in (result[1] or {}).items():
-                row['election_title_{}'.format(locale)] = (title or '').strip()
+            for locale in Election.i18n_used_locales:
+                title = result[1] and result[1].get(locale, '') or ''
```

Comment:
Slightly better: `title = result[1] and result[1].get(locale) or ''`

Reply:
yep =)